core/CMakeLists.txt

```diff
@@ -13,10 +13,10 @@ include(SetROOTVersion)
 # Update etc/gitinfo.txt for every build.
 add_custom_target(gitinfotxt
   ALL
-  COMMAND ${CMAKE_COMMAND} -DSRCDIR=${CMAKE_SOURCE_DIR} -DBINDIR=${CMAKE_BINARY_DIR} -P ${CMAKE_SOURCE_DIR}/cmake/modules/UpdateGitInfo.cmake
+  COMMAND ${CMAKE_COMMAND} -DSRCDIR=${CMAKE_SOURCE_DIR} -DBINDIR=${CMAKE_BINARY_DIR} -DINSTALL_DIR=${CMAKE_INSTALL_PREFIX} -P ${CMAKE_SOURCE_DIR}/cmake/modules/UpdateGitInfo.cmake
```

Comment:
Instead of using the script to write the file, shouldn't we simply use the `installl()` CMake function to install it?